src/main/scala/PrivateHome/Devices/MQTT/mqttClient.scala

```diff
@@ -0,0 +1,62 @@
+package PrivateHome.Devices.MQTT
+
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence
+import org.eclipse.paho.client.mqttv3.{MqttClient, MqttMessage, _}
+
+object mqttClient {
+
+  private val brokerUrl = "tcp://localhost:429"
+  private val home = "Home/#"
+  private val persistence = new MemoryPersistence
+  private val client = new MqttClient(brokerUrl, MqttClient.generateClientId, persistence)
+
+  var lastMsg: String = "" //here you can get the last recieved message
+  var lastID: String = "" //her you can get the last recieved id
+
+  client.connect
+  client.subscribe(home)
+
+  val callback = new MqttCallback {
+
+    /**
+     * This method acts after recieving an MQTT-Message
+     *
+     * @param topic   is the topic in which the message was recieved. Mostly it contains the id, which you can see better in lastID.
+     * @param message is the message which was recieved. You can also get it from lastMsg.
+     */
+    override def messageArrived(topic: String, message: MqttMessage): Unit = {
+      println("Receiving Data, Topic : %s, Message : %s".format(topic, message))
+      lastMsg = message.toString
+      lastID = ""
+      lastID = topic.substring(topic.length - 5)
```

Comment:
What happends if the Topic is home/cmd/hiabcde

Reply:
what is with the cmd that means this is a cmd not an State change or should this be the same?